src/databaseaccessor/commentdb.cpp

```diff
@@ -0,0 +1,113 @@
+#include "commentdb.h"
+#include <QDebug>
+CommentDb::CommentDb(): Accessor()
+{
+}
+
+void CommentDb::addCommentsToDb(const QVector<Comment> &comments)
+{
+    for (auto &i : comments)
+    {
+        execQuery(addCommentQuery(i));
+    }
+    query.finish();
+}
+
+void CommentDb::deleteCommentFromDb(const int commentLine, const QString commentFile)
+{
+    execQuery(deleteCommentQuery(commentLine, commentFile));
+    query.finish();
+}
+
+void CommentDb::deleteCommentsFromDb(const QString commentFile)
+{
+    execQuery(deleteAllCommentsInFileQuery(commentFile));
+}
+
+QVector<Comment> CommentDb::getAllCommentsFromFile(const QString filename)
+{
+      execQuery(numberOfCommentInFileQuery(filename));
+      query.first();
+      int count_of_messages =query.value(0).toInt();
+      QVector<Comment> comments(count_of_messages);
+      execQuery(allCommentInFileQuery(filename));
+      int counter = 0;
+      while (query.next()) {
+          fillStructComment(comments[counter]);
+          counter++;
+      }
+      query.finish();
+      for(auto i : comments){
+          qDebug()<<i.mLine<<" "<<i.mText;
+      }
+      return comments;
+}
+
+Comment CommentDb::getCommentFromDb(const int commentLine, const QString commentFile)
+{
+    execQuery(getCommentQuery(commentLine, commentFile));
+    query.first();
+    Comment rComment;
+    fillStructComment(rComment);
+    query.finish();
+    return rComment;
+}
+
+QString CommentDb::addCommentQuery(const Comment &comment)
+{
+    return "INSERT INTO Comment (line, idFile, idUser, text) VALUES ('"
+            +QString::number(comment.mLine) + "', "
+            "(SELECT id FROM File WHERE name = '" + comment.mFile + "'), "
+            "(SELECT id FROM User WHERE nickname = '" + comment.mUser + "'), '"
+            +comment.mText + "')" ;
+}
```

Comment:
It might be helpful to use methods: QSqlQuery:: bindValue or QString::arg in order to avoid concatenation 

E.g. 
QString status = QString("Processing file %1 of %2: %3").arg(i).arg(total).arg(fileName);

QSqlQuery query;
query.prepare("INSERT INTO person (id, forename, surname) "
"VALUES (:id, :forename, :surname)");
query.bindValue(":id", 1001);
query.bindValue(":forename", "Bart");
query.bindValue(":surname", "Simpson");